app/controllers/home_controller.rb

```diff
@@ -0,0 +1,3 @@
+class HomeController < ApplicationController
+  def index; end
```

Comment:
redirect to the login page if the user is not signed in

Reply:
Done